packages/bundles/kairos-overlay-files/files/system/oem/11_persistent.yaml

```diff
@@ -11,3 +11,6 @@ stages:
         expand_partition:
           # Size 0 is required to specify all remaining space
           size: 0
+      commands:
+        # grow filesystem
+        - resize2fs /dev/disk/by-label/COS_PERSISTENT
```

Comment:
shouldn't we call also growpart first? `resize2fs` just enlarges the fs IIRC

Reply:
that what we want no? The partition is already resized but the FS is not, so its only taking a small part of the partition. We only want to resize the fs, the expansion is done by yip. In fact this is a workaround, the real fix should go into the layout plugin to call the fs expansion after the partition recreation to not leave this like this.

Both growpart and resize2fs work the same, they resize the FS. Not sure if growpart is already installed in our images but resize2fs is, as its part of e2fsprogs

Reply:
growpart resizes a partition to my knowledge, and NOT the filesystem. resize2fs resizes the FS, NOT the partition (they are doing two different things).

I think to make things more simpler, if we need many workaround we can just drop the usage of the cloud config here and just use growpart+resize2fs which works for both MBR and GPT. 

There is one subtle issue with this too which is this: https://github.com/kairos-io/kairos-agent/blob/307eabbb36c9c6969681f4a3852e53d4dd572a2e/pkg/partitioner/disk.go#L359 I've been facing this myself - and the result was unpleasant. If we try to extend the partitions regardless on each boot, there was a change that the partition was completely nuked between standard reboots